File: app/models/mit.rb
class Mit < ActiveRecord::Base
  acts_as_taggable

  validates_presence_of :title

  validates_length_of :description, maximum: 500

  has_many :comments

  belongs_to :user


  def niceDate
    created_at.strftime("%m-%d-%Y")
  end

end

File: app/assets/javascripts/models/mit.js
App.Mit = DS.Model.extend({
  title: DS.attr(),
  created_at: DS.attr(),
  description: DS.attr(),
  comments: DS.hasMany('comment'),
  complete: DS.attr(),
  tagList: DS.attr(),
  tags: DS.hasMany('tag'),
  user: DS.belongsTo('user'),

  date: function(){
     return moment(this.get('created_at')).format("MM-DD-YYYY");
  }.property('created_at')



});

